devicon.json

```diff
@@ -7309,6 +7309,26 @@
         "color": "#903a36",
         "aliases": []
     },
+    {
+        "name": "hibernate",
```

Comment:
The object needs to be moved so it's in alphabetical order in relation to the other icons